File: src/2980.check-if-bitwise-or-has-trailing-zeros.3246/solution.ts
function hasTrailingZeros (nums: number[]): boolean {
    let has = false;
    for (let i = 0; i < nums.length; i++) {
        if (nums[i] % 2 === 0) {
            if (has) {
                return true;
            }
            has = true;
        }
    }
    return false;
}

File: src/2980.check-if-bitwise-or-has-trailing-zeros.3246/solution.java
class Solution {
    public boolean hasTrailingZeros(int[] nums) {
        boolean hasEven = false;
        for (int n : nums) {
            if (n % 2 == 0) {
                if (hasEven) {
                    return true;
                }
                hasEven = true;
            }
        }
        return false;
    }
}